csp_collector.go

```diff
@@ -77,6 +77,9 @@ func validateViolation(r CSPReport) error {
 		"chromenull://",
 		"resource://",
 		"about:blank",
+		"android-webview",
+		"ms-browser-extension",
+		"http://adblockers.opera-mini.net",
```

Comment:
I'm a little unsure about this one since it's not too well documented on the web. Is there any more information available on this one?